com.unity.render-pipelines.universal/Runtime/Passes/DrawSkyboxPass.cs

```diff
@@ -68,7 +70,26 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
             else
 #endif
             {
-                context.DrawSkybox(renderingData.cameraData.camera);
+                if ((DebugHandler != null) && DebugHandler.IsActiveForCamera(ref renderingData.cameraData))
+                {
+                    CommandBuffer cmd = CommandBufferPool.Get();
+
+                    foreach (DebugRenderSetup debugRenderPass in DebugHandler.CreateDebugRenderSetupEnumerable(context, cmd))
```

Comment:
Looks like `debugRenderPass` is not being used anywhere? Should we remove it?

Reply:
You're right, the loop was unnecessary. Removed it and moved the check to the top of the function, so it applies to the XR codepath as well.